src/main/host/host.rs

```diff
@@ -188,10 +188,7 @@ impl crate::utility::IsSend for Host {}
 // TODO: use derive(Debug) if/when all fields implement Debug.
 impl std::fmt::Debug for Host {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        f.debug_struct("Host")
-            .field("chost", &self.chost)
-            .field("info", &self.info)
-            .finish()
+        f.debug_struct("Host").field("info", &self.info).finish()
```

Comment:
nit: maybe `finish_non_exhaustive` instead since there are fields we aren't printing?

Reply:
done